README.md

```diff
@@ -395,6 +401,123 @@ ActiveSupport::Notifications.subscribe(/rack_attack/) do |name, start, finish, r
 end
 ```
 
+## Fault Tolerance & Error Handling
+
+Rack::Attack has a mission-critical dependency on your [cache store](#cache-store-configuration).
+If the cache system experiences an outage, it may cause severe latency within Rack::Attack
+and lead to an overall application outage.
+
+This section explains how to configure your application and handle errors in order to mitigate issues.
+
+### Built-in error handling
+
+By default, Rack::Attack "does the right thing" when errors occur:
+
+- If the error is a Redis or Dalli cache error, Rack::Attack allows the error and allow the request.
```

Comment:
Small point: 

> Rack::Attack allows the error and allow the request

should that be something more like

> Rack::Attack catches and logs the error and allows the request

Reply:
Changed.